osx/post_installation.sh

```diff
@@ -124,7 +124,19 @@ if [ ! -d "$LAUNCH_AGENTS" ]; then
     fi
 fi
 
+
+((STEP++))
+echo "$STEP/$STEPS. Create plist in /Library/LaunchAgents folders..."
+if [ ! -d "$LAUNCH_AGENTS" ]; then
```

Comment:
We need this check if we are using `~/Library/LaunchAgents/` but I think there's no need for this if we are using `/Library/LaunchAgents/`.